### PR TITLE
add pinchBeetle Tweek

### PR DESCRIPTION
**Pull request by ghost**
I made PinchBeetle Tweek

Changed:
When Pinched,you cannot guard
when guard success,pinchBeetle is confused

### CODE REVIEW

File: src/main/java/twilightforest/client/renderer/entity/RenderTFPinchBeetle.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	public RenderTFPinchBeetle(RenderManager manager, ModelBase par1ModelBase, float shadowSize) {
+		super(manager, par1ModelBase, shadowSize);
+	}
+
+	@Override
+	public void doRender(EntityTFPinchBeetle entity, double x, double y, double z, float entityYaw, float partialTicks) {
+		super.doRender(entity, x, y, z, entityYaw, partialTicks);
+		if (!Minecraft.getMinecraft().isGamePaused() && entity.isConfuse()) {
+			Vec3d pos = new Vec3d(entity.posX, entity.posY + 1.0D, entity.posZ).add(new Vec3d(1.0D, 0, 0).rotateYaw((float) Math.toRadians(entity.getRNG().nextInt(360))));
+			Minecraft.getMinecraft().world.spawnParticle(EnumParticleTypes.CRIT, pos.x, pos.y, pos.z, 0, 0, 0);
```
Comment (Tamaized) [inline src/main/java/twilightforest/client/renderer/entity/RenderTFPinchBeetle.java]: Why make this a render, why not just do it within the entity's tick method. Generally a bad idea to spawn particles inside renders anyhow due to framerate

Edit: Ah i see you copied from the naga renderer, not sure why i did that in the first place. It's an issue nonetheless

Comment (ghost) [reply src/main/java/twilightforest/client/renderer/entity/RenderTFPinchBeetle.java]: oh sorry...
I will fix it

**Comment by ghost** [comment]
I fixed render proglem

File: src/main/java/twilightforest/client/TFClientProxy.java [hunk 2]
```diff
@@ -128,7 +128,6 @@ public void preInit() {
 		RenderingRegistry.registerEntityRenderingHandler(EntityTFHarbingerCube.class, RenderTFHarbingerCube::new);
 		RenderingRegistry.registerEntityRenderingHandler(EntityTFAdherent.class, m -> new RenderTFAdherent(m, new ModelTFAdherent(), 0.625F, "adherent.png"));
 		RenderingRegistry.registerEntityRenderingHandler(EntityTFRovingCube.class, RenderTFRovingCube::new);
-		RenderingRegistry.registerEntityRenderingHandler(EntityTFRisingZombie.class, m -> new RenderTFBiped<>(m, new ModelTFRisingZombie(), 0.5F, "textures/entity/zombie/zombie.png"));
```
Comment (Tamaized) [inline src/main/java/twilightforest/client/TFClientProxy.java]: ?

Comment (ghost) [reply src/main/java/twilightforest/client/TFClientProxy.java]: ahh...

**Comment by ghost** [comment]
okey, fixed my mistake

**Pull request closed without merging**